protocol.md

```diff
@@ -110,13 +110,11 @@ If any of these check fail, the connection is closed.
 
 Prerequisite: the *user* has created a [Verification Key].
 
-Except for the first *device*, which is validated in a specific way previously described, additional *device*s must be validated by an already registered *device*. In practice, these *device*s are validated by the *ghost device*.
+Except for the first *device*, which is validated in a specific way previously described in the [Sign up process](#sign-up), additional *device*s must be validated by an already registered *device*. In practice, these *device*s are validated by the *virtual device*.
 
 Given the *user*'s [Verification Key], the steps to register a new *device* are as follows:
 
-1. Extract the *ghost device*'s [Device Encryption Key Pair] and [Device Signature Key Pair] from the [Verification Key]
-2. Pull the *Trustchain* up to the *ghost device*'s `device_creation` *block*, verify it and extract the [User Encryption Key Pair] from it
-3. Decrypt the [User Encryption Key Pair] using the  *ghost device*'s private [Device Encryption Key Pair]
```

Comment:
you can't skip this, you need the user encryption key pair to make the device block